PhysicsTools/HepMCCandAlgos/python/genPUProtons_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+genPUProtons = cms.EDProducer("GenPUProtonProducer",
+	mix = cms.string("mix"),
+	bunchCrossingList = cms.vint32(0),
+	minPz = cms.double( 4800. )
```

Comment:
the value should be changed so that either one value fits all [reasonable] data taking that we had or modified by era